helper_test.go

```diff
@@ -557,6 +553,22 @@ func createAppliedRun(t *testing.T, client *Client, w *Workspace) (*Run, func())
 	return createRunWithStatus(t, client, w, 90, RunApplied)
 }
 
+func plannedRunStatus() RunStatus {
+	if paidFeaturesDisabled() {
```

Comment:
Since you technically could have paid features without cost estimation enabled, and because you're checking the state of the run specifically in `hasApplyableStatus`, I'd consider changing this to be `if r.CostEstimate != nil` or similar, but I guess that would require some reworking of `createRunWithStatus` as you'll need some sort of argument to say "Planned, whatever 'planned' means in the context of the run". 

It almost seems like the API here should allow for 'statuses' to adverbs not directly tied to specific run states (e.g. "applyable"), necessarily - because the state changes depending on the nature of the workspace/run (what features are enabled, etc).

Reply:
Very good point! The `hasApplyableStatus` helper should certainly be able to make the distinction between paid features enabled in general and cost estimation enabled specifically, since we have access to the run there.

It seems more reasonable for the `createPlannedRun` helper to continue to expect different statuses depending on whether paid features are enabled - if in the future someone wants to test with cost estimation disabled, they can use `createRunWithStatus` by itself.